src/selectedtests/task_mappings/task_mappings_cli.py

```diff
@@ -32,7 +30,7 @@ def _setup_logging(verbose: bool):
 def cli(ctx, verbose: bool):
     """Entry point for the cli interface. It sets up the evg api instance and logging."""
     ctx.ensure_object(dict)
-    ctx.obj["evg_api"] = RetryingEvergreenApi.get_api(use_config_file=True)
+    ctx.obj["evg_api"] = get_evg_api()
```

Comment:
We use environment-variable-based Evergreen API authentication elsewhere in the service so I updated the CLIs to use it too